releasenotes/notes/deprecate-complex-amp-old-qpy-d3d183e84c395d9c.yaml

```
@@ -0,0 +1,13 @@
+---
+deprecations:
+  - |
+    Loading library :class:`~.qiskit.pulse.ScalableSymbolicPulse` objects with
+    complex ``amp`` parameter from
+    qpy files of version 5 or lower (Qiskit Terra < 0.23.0) is now deprecated.
+    Following the deprecation in Qiskit 1.0.0, complex `amp` will be automatically
```

I think that this should be:
```suggestion
    Following the removal in Qiskit 1.0.0, complex `amp` will be automatically
```
because the conversion hasn't been implemented yet, right? I just quickly checked #10357 and didn't see it, but I might be wrong.

---

That's right. Thanks.